### ISSUE TITLE
Use loom correctly

### ISSUE THREAD

**Issue by Gamebuster19901** [WilderForge/WilderWorkspace#20]
Right now, jar remapping is kinda hardcoded... Mods depending on remapped jars won't see any changes other mods make (ex: changes made via mixins/accesstransfromers).

The solution is to set up and configure the loom gradle plugin properly.

**Comment by Gamebuster19901** [comment]
This will also help wiht #6

**Comment by Gamebuster19901** [comment]
Blocked by #23

**Comment by Gamebuster19901** [comment]
Superseded by #24
